pump/storage/storage.go

```diff
@@ -686,6 +688,15 @@ func (a *Append) doGCTS(ts int64) {
 			return
 		}
 
+		failpoint.Inject("l0-file-too-much", func() {
+			if l0TestLarge < l0Trigger {
+				l0TestLarge++
+			} else {
+				l0TestLarge -= l0Trigger
+			}
+			l0Num = l0TestLarge
+		})
```

Comment:
Can you explain what cases are setup here?

Reply:
Please add test to check that the waiting happens when the number of l0 files is larger than l0Trigger.
And mock the wait time below to make it shorter.